internal/kubernetes_client/kubernetes_client.go

```diff
@@ -14,7 +14,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-package policy
+package kubernetes_client
```

Comment:
Why not "kubernetes"?

Reply:
I think that at some point during refactor, there were linter complaints about collisions between variable names and package names.